src/OrbitQt/include/OrbitQt/orbitsamplingreport.h

```diff
@@ -45,7 +45,9 @@ class OrbitSamplingReport : public QWidget {
   void on_PreviousCallstackButton_clicked();
   void OnCurrentThreadTabChanged(int current_tab_index);
 
- private:
+  // TODO(https://github.com/google/orbit/issues/4589): Remove redundant "private" once slots is not
```

Comment:
Should the `private slots:` above also have the relevant TODO like in the other files?

Reply:
Done.